### ISSUE TITLE
custom styles for the toolbar

### ISSUE THREAD

**Issue by magician11** [brijeshb42/medium-draft#19]
We're wanting to add colour inline styles to the toolbar.

I've been reading [the docs](https://github.com/brijeshb42/medium-draft/wiki/Customising-the-Toolbar) and going through the code, but I'm not completely clear how to create a custom style.. with I assume `customStyleMap`.

Ideally it would be great to add something like [react-color](https://casesandberg.github.io/react-color/).

Do we use [customrenderer](https://github.com/brijeshb42/medium-draft/blob/master/src/components/customrenderer.js) somehow?

Appreciate your guidance.

**Comment by brijeshb42** [comment]
You can use `customrender` for this purpose but I am afraid, right now, the toolbar does not have a lot of customizability besides controlling the order of the buttons.

**Comment by magician11** [comment]
Ok, so if I wanted to set some text to one single colour that might work. But probably not a palette of colours for now?

**Comment by brijeshb42** [comment]
Yes. You got it right. Buttons that require more than one interaction aren't there. (One is the Link button but that is hardcoded). I will have to think of something like the `sideButtons` where devs have full control of how the button looks and what it does.
